src/pybind/mgr/dashboard/frontend/src/styles/_carbon-defaults.scss

```
@@ -149,3 +149,12 @@ Code snippet
 .cds--snippet {
   width: fit-content;
 }
+/******************************************
+Carbon Popover
+******************************************/
+.cds--popover-content {
+  padding: 1rem;
+  background-color: theme.$layer-02;
+  max-height: 200px;
+  overflow:auto
```

let's increase the width as well
```suggestion
  overflow:auto;
  max-inline-size: 30rem;
```

---

I think you'll have to also add
```
.cds--toggletip-content {
  max-inline-size: 30rem;
}
```

but you can check that

---

I guess its not required... becoz its not making any impact after adding